# Fix zlib target.
#
# ```
# sed --in-place \
#     -e 's/hdrs = \[\"zlib\.h\"\],/hdrs = ["zconf.h", "zlib.h"],/g' \
#     "third_party/zlib.BUILD"
# ```

diff --git third_party/zlib.BUILD third_party/zlib.BUILD
index b8464cf2db..f8d48f77d3 100644
--- third_party/zlib.BUILD
+++ third_party/zlib.BUILD
@@ -31,7 +31,7 @@ cc_library(
         "zutil.c",
         "zutil.h",
     ],
-    hdrs = ["zlib.h"],
+    hdrs = ["zconf.h", "zlib.h"],
     copts = select({
         "@xla//xla/tsl:windows": [],
         "//conditions:default": [
